### DISCUSSION TITLE
Duda RangeTree y funcionalidad IN-X-RANGE

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#67] [Tarea 1] [answered]
Tengo una duda respecto a la funcionalidad IN-X-RANGE y su relación con RangeTree, ya que en el pdf de la tarea se pide que obtengamos todos los elementos desde A (incluyendo A) hasta B (sin incluir B), pero en la página de WikiPedia de RangeTree se menciona que RangeTree obtiene los valores incluyendo a A e incluyendo a B. Necesitamos sacar manualmente el caso de B o es esto un error del enunciado?

**Comment by benjavicente** [comment, marked as answer]
El enunciado está bien. El caso que muestra Wikipedia es considerando el rango inclusive, no es una limitación de la EDD.

**Reply by MaxAl100** [reply]
Yap, muchas gracias!
